build/ci.go

```diff
@@ -322,6 +325,29 @@ func doTest(cmdline []string) {
 	build.MustRun(gotest)
 }
 
+func downloadSpecTestFixtures(cachedir string) string {
+	// Download the execution-spec-tests
+	const version = "0.2.4"
+
+	csdb := build.MustLoadChecksums("build/checksums.txt")
+	ext := ".tar.gz"
+	if runtime.GOOS == "windows" {
+		// TODO (MariusVanDerWijden) enable windows tests once spec tests provide a .zip
+		return ""
+		ext = ".zip"
+	}
```

Comment:
Why can't windows use the `.tar.gz` package?

Reply:
I thought that windows can't do .tar.gz. but it looks like I'm wrong there. I haven't used windows in years